org.openhab.binding.zigbee/ESH-INF/thing/channels.xml

```diff
@@ -146,6 +146,15 @@
         <state readOnly="true" />
     </channel-type>
 
+    <!-- IAS Tampering Switch Channel -->
```

Comment:
Minor comment: To align with other comments, write `IAS Tamper Channel` instead of `IAS Tampering Switch Channel`.